hw/top_earlgrey/data/ip/chip_entropy_src_testplan.hjson

```diff
@@ -179,5 +179,74 @@
       tests: []
       bazel: []
     },
+    {
+      name: chip_sw_entropy_src_fw_observe_many_contiguous
+      desc: '''Verify that the software can observe samples fast enough to extract many contiguous samples.
+
+            Procedure:
+            - Disable the entropy complex.
+            - Configure realistic health test threshold values for FIPS/CC compliant mode.
+            - Enable ENTROPY_SRC in FIPS mode (FIXME: is it necessary to use FIPS mode?).
+            - Enable Firmware Override - Observe.
+            - Collect 1000 samples from the observe FIFO:
+              - Verify after reading each sample that the observe FIFO overflow status has not been set.
+              - FIXME: should the SW run a health check to make sure that it is fast enough?
+
+            Tests:
+            - The procedure must be run once with ENTROPY_SRC.RNG_BIT_ENABLE disabled (4-bit / sample) and
+              four times with the ENTROPY_SRC.RNG_BIT_ENABLE feature being enabled (once for every channel).
+
+            Notes for silicon targets:
+            - This test requires access to the ENTROPY_SRC.FW_OV.OBSERVE feature, which is gated by OTP.
+              This test might thus need to be done in a secure facility.
+            '''
+      features: [
+        "ENTROPY_SRC.MODE.FIPS", // FIXME: maybe
+        "ENTROPY_SRC.FW_OV.OBSERVE"
+        "ENTROPY_SRC.RNG_BIT_ENABLE",
+      ]
+      stage: V2
+      si_stage: SV3
+      lc_states: ["TEST_UNLOCKED", "PROD"]
+      tests: []
+      bazel: []
+    },
+    {
+      name: chip_sw_entropy_src_fw_extract_and_insert
+      desc: '''Verify that the software can observe samples fast enough to extract many contiguous samples.
+
+            Procedure:
+            - Disable the entropy complex.
+            - Configure realistic health test threshold values for FIPS/CC compliant mode.
+            - Enable ENTROPY_SRC in FIPS mode (FIXME: is it necessary to use FIPS mode?)
+            - Enable Firmware Override - Extract & Insert.
+            - Enable CSRNG.
+            - Enable both EDNs in auto request mode.
+            - Trigger the execution of various cryptographic hardware blocks consuming entropy to stress test the entropy complex.
+            - Repeat until all entropy consuming endpoints have finished their operations:
+              - Read a sample from the observe FIFO.
+              - Verify that the observe FIFO overflow status has not been set.
+              - Insert the sample back into the entropy flow.
```

Comment:
Also here, we need to read contiguous windows of at least 1000 samples. Ideally, the test reads more than 2 windows and tests window sizes of 1k and 4k samples.

So when doing 1k windows, the procedure would be:
- Configure ([OBSERVE_FIFO_THRESH](https://opentitan.org/book/hw/ip/entropy_src/doc/registers.html#observe_fifo_thresh)) to a sensible value.
- Repeat until all entropy consuming endpoints have finished their operations: 
              - Read a 1k sample window from the observe FIFO.
              - Verify that the observe FIFO overflow status has not been set.
              - Insert the 1k samples back into the entropy flow.
              - Let the observe FIFO overflow.
              - Empty the observe FIFO and discard any values read.

Note that depending on whether the conditioner is used a different amount of entropy is required per seed. If the conditioner is not used, inserting 384 bits should give one seed.